CorsixTH/Lua/room.lua

```diff
@@ -492,16 +497,16 @@ end
 
 --! When a valid member of staff enters the room this function is called.
 -- Can be extended in derived classes.
---!param humanoid The staff in question
---!param already_initialized If true, this means that the staff has already got order
+--!param staff (object) The staff in question
```

Comment:
```suggestion
--!param staff (Staff) The staff in question
```
There is a `Staff` base class.